tooling/template-base/components/Navigation/MobileMenu.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import avatar from '../../renderer/avatar.png';
+
+const MobileMenu = () => {
+  return (
+    <div className="">
+      <ul className="space-y-2">
+        <div className="flex py-5">
+          <div className="h-11 w-11">
+            <img src={avatar} />
+          </div>
+          <div className="mx-3 my-auto">
+            <button className="hover:text-blue-700">
+              <a href="/sign-in">My Profile</a>
```

Comment:
Think it'd be weird for to direct users clicking "My Profile" to a sign in page. If they're already signed in, we should direct them to `/learn/profile`. Which brings me to another point - with this and Line 29 in this component, there's an assumption that the user is signed in, which may not be the case. You'll want to pass the `CurrentUser` down through props or use `usePageContext` to access the current user, and if there is one then conditionally render this and the "Sign Out" button. Would recommend checking out and implementing in the other NavBars as well.

Reply:
what link would I use for the help page?

Reply:
"and if there is one then conditionally render **this** and the "Sign Out" button."
what is "this" referring to?